src/Uno.UI/UI/Xaml/Style/Generic/Generic.Native.xaml

```diff
@@ -15,8 +15,17 @@
 					xmlns:toolkit="using:Uno.UI.Toolkit"
 					xmlns:maps="using:Windows.UI.Xaml.Controls.Maps"
 					xmlns:mapsPresenter="using:Windows.UI.Xaml.Controls.Maps"
+					xmlns:wasm="http://uno.ui/wasm"
 					xmlns:not_wasm="http://uno.ui/not_wasm"
-					mc:Ignorable="d xamarin ios android not_wasm">
+					mc:Ignorable="d xamarin ios android wasm not_wasm">
+
+	<wasm:ResourceDictionary.MergedDictionaries>
+		<!--
+			On WASM we don't  necessarilly have native style, so we refrence the Windows styles as "native" so we are able
+			to multi-target their usage in application without having to discover which native styles are available or not.
```

Comment:
This seems reasonable but it should ideally be documented somewhere. (Generic.Native.xaml doesn't count as 'somewhere.')